### PR TITLE
[T4A5][T10-B1]KelvinNg

### PR DESCRIPTION
**Pull request by kelvinnpc**

### CODE REVIEW

**Comment by kelvinnpc** [comment]
Ready for review

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    		this.action = "-";
+    	}
+    	
+    	this.name = person.getName().toString();
+    	this.tag = tag.toString();
+    }
+    
+    @Override
+    public String toString() {
+    	return action + " " + name + " " + tag;
```
Comment (nirandiw) [inline src/seedu/addressbook/data/tag/Tagging.java:27]: Wasn't there suppose to be a square bracket around the tag?

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+/*
+ * Represents the association between Person and Tag
+ */
+public class Tagging {
+    private String action;
```
Comment (nirandiw) [inline src/seedu/addressbook/data/tag/Tagging.java:10]: I can't agree that this is an Association Class. Why? you are not maintaining the Person and Tag associated with the tagging. 
Hints: Maintain the Person and the Tag class in the Tagging class cited to Strings name, tag

It is not a good idea to use String type to maintain the "action". Consider a boolean or an enum.

Where are you maintain the list of tagging?

**Comment by nirandiw** [comment]
@kelvinnpc Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**